docs/data/sensitive-data/sd-desktop-secondary-working.md

```diff
@@ -0,0 +1,145 @@
+  
+- **File structure**: Instead of “My Documents” or drives labeled C: or D:, Linux has a different way of organizing files. The main directory starts with /, and you’ll see folders like /home for your personal files.
+  
+- **Benefits**: Linux is very stable and less likely to crash, making it ideal for long or intensive computing tasks.
+
+## Security-related features and limitations
+
+SD Desktop is a secure environment designed specifically for analysing sensitive data and the services has certain features that may work differently than a regular computer. For each of these features, a specific step-by-step guide is available below. If you are using the service for the first time, reviewing these guides will help you become familiar with how everything works.
+
+- **Isolated from the internet**: your virtual desktop is completely isolated from the internet. This means that although you can open a web browser like Firefox, you will not be able to access websites or online repositories directly. This feature helps keep your data safe by reducing the risk of online threats.
+  
+- **Encrypted file only**: Only encrypted files are accessible from your virtual desktop. All files exported from the virtual desktop also need to be encrypted. This adds an extra layer of protection for your data.
+  
+- **Controlled file access with Data Gateway**: The import of data and software is restricted in SD Desktop. You cannot import any data or software yourself for security reasons. If you are working with a dataset for which you have received a permit from a data controller, the only way to access the data for analysis is by utilizing a specific application called [Data Gateway](./sd-desktop-secondary-access.md) and imported only via the SD Apply service (upon approval for reuse).
+
+- **Data export from SD Desktop is restricted**. Only *non-sensitive* results can be exported from the workspace, and those can only be exported by the CSC specialitis. Instructions for exporting your results are provided in the next section.
+  
+- **Limited storage space**: The virtual desktop is designed primarily for data analysis and has limited storage space. To expand storage, you can add an external volume (such as an external hard drive) during the desktop setup. This external volume is accessible to all project members and also serves as a backup for imported data.
+  
+- **Open source software only**: Only open-source software can be installed in the virtual desktop, as it does not currently support licensed or proprietary software. Each virtual desktop comes with a default set of pre-installed software, but you can customise it further using an easy-to-use application or, for advanced users, through Apptainer. If the software you need is not listed below, please [contact CSC Service Desk](../../support/contact.md) for support (subject: SD Desktop).
```

Comment:
I think " but you can customise it further using an easy-to-use application or, for advanced users, through Apptainer. If the software you need is not listed below, please [contact CSC Service Desk](../../support/contact.md) for support (subject: SD Desktop)" should also be changed to something like in the previous mention about software.